CHANGES_NEXT_RELEASE

```diff
@@ -7,3 +7,4 @@
 - Fix: Service-Path supported for forwarding of registrations between Config Manager and Context Broker (Issue #719)
 - Fix: Service-Path supported for forwarding to Context Providers (Issue #719)
 - Fix: Hash sign (#) in service-path, or multiple service-paths not valid for updateContext requests (Issue #746)
+- Fix: Not forwarding the default Service-Path to context providers (Issue #714)
```

Comment:
Maybe we need a .test (in the 714_* directory) to ensure that service path is not being forwarded in this case?